File: app/src/main/java/eu/toma/dev/playground/warcraft/model/item/RequiredAbility.java
package eu.toma.dev.playground.warcraft.model.item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RequiredAbility {

    @SerializedName("spellId")
    @Expose
    public Integer spellId;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("description")
    @Expose
    public String description;

}

File: app/src/main/kotlin/warcraft/model/item/RequiredAbility.kt
package warcraft.model.item

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RequiredAbility {

    @SerializedName("spellId")
    @Expose
    var spellId: Int? = null
    @SerializedName("name")
    @Expose
    var name: String
    @SerializedName("description")
    @Expose
    var description: String

}